bindings/python/iota_sdk/types/output.py

```diff
@@ -110,16 +104,15 @@ class AccountOutput:
             decoder=deserialize_unlock_conditions
         ))
     features: Optional[List[Union[SenderFeature,
-                            MetadataFeature]]] = field(default=None,
-                                                       metadata=config(
-                                                           decoder=deserialize_features
-                                                       ))
+                            MetadataFeature, NativeTokenFeature]]] = field(default=None,
```

Comment:
Should be in the Foundry instead of Account?

Reply:
Yes good catch